README.md

```diff
@@ -451,6 +451,46 @@ unsubscribe() // stops listening
 stelace.auth.getTokens(data, [options], [callback])
 
 
+### Forward methods (advanced usage)
+
+The below methods are provided to hit custom endpoints with the Stelace authentication headers.
+This is especially useful if one needs to add custom logic in serverless systems like lambda functions.
```

Comment:
It could sound more personal. I suggest:

```
Following methods are provided to hit custom endpoints with Stelace authentication headers.
This is especially useful if you need to call external endpoints with custom logic, e.g. using serverless lambda functions.
```